build/storage/tests/it/ptf_tests/tests/test_hot_plug_recipe.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os
+from ptf.base_tests import BaseTest
+
+from system_tools.config import (
+    StorageTargetConfig, IPUStorageConfig, HostTargetConfig,
+)
+from system_tools.ssh_terminal import SSHTerminal
+from system_tools.test_platform import HostPlatform
+from system_tools.services import (
+    CloneRepository, RunStorageTargetContainer, RunIPUStorageContainer, RunHostRargetContainer
+)
+
+
+class TestHostPlatform(BaseTest):
+
+    def setUp(self):
+        self.storage_target_terminal = SSHTerminal(StorageTargetConfig())
+        self.ipu_storage_terminal = SSHTerminal(IPUStorageConfig())
+        self.host_target_terminal = SSHTerminal(HostTargetConfig())
+
+        self.storage_target_platform = HostPlatform(self.storage_target_terminal)
+        self.ipu_storage_platform = HostPlatform(self.ipu_storage_terminal)
+        self.host_target_platform = HostPlatform(self.host_target_terminal)
+
+    def runTest(self):
+        self.storage_target_platform.host_system_setup()
+        self.ipu_storage_platform.host_system_setup()
+        self.host_target_platform.host_system_setup()
+
+    def tearDown(self):
+        pass
+
+
+class TestDeployContainers(BaseTest):
+
+    def setUp(self):
+        self.storage_target_terminal = SSHTerminal(StorageTargetConfig())
+        self.ipu_storage_terminal = SSHTerminal(IPUStorageConfig())
+        self.host_target_terminal = SSHTerminal(HostTargetConfig())
+
+        self.storage_target_terminal.delete_all_containers()
```

Comment:
for now it's not necessary created a lot of classes - empty classes , if we will have a few methods we will split terminal class for other, terminal not will be big because we decided to use more raw paramiko so we not need big classes like previous, now we should keep it simple and develop in the future